src/imap.rs

```diff
@@ -2076,6 +2085,21 @@ pub async fn get_config_last_seen_uid(context: &Context, folder: &str) -> Result
     }
 }
 
+/// Whether to ignore fetching messages from a folder.
+///
+/// This caters for the [`Config::OnlyFetchMvbox`] setting which means mails from folders
+/// not explicitly watched should not be fetched.
```

Comment:
```suggestion
/// Whether to ignore messages from a folder, i.e. skip fetching from the folder.
///
/// This is needed for the [`Config::OnlyFetchMvbox`] setting.
```

"is needed" is a simpler expression that "cater" which I had to look up :)